### PR TITLE
Fixed IE 9 & IE 10 issues

### PR DESCRIPTION
**Pull request by atuggle**
- Added import "babel-polyfill"; for the React app
- Updated crt-start page to be more compliant with IE 9 & 10 issues:
  - It didn't like <path />...  changed to <path ...></path>
  - It didn't like 'const' or 'let' so I changed them to var

## Additions
- Added two more bug fixes to this (both are very small)
  - GH#114
  - GH#126

## Review

- @dcmouyard 

[Preview this PR without the whitespace changes](?w=0)

### CODE REVIEW

**Comment by dcmouyard** [comment]
@atuggle I’m seeing the following build error: 
```
Module not found: Error: Can't resolve 'babel-polyfill' in '/cfgov-refresh/develop-apps/teachers-digital-platform/teachers_digital_platform/crtool/src'
```

**Comment by atuggle** [comment]
@dcmouyard You need to run ./setup.sh  or npm install

its a new node_module that needs to get installed.  It appears to be working on the build server

File: teachers_digital_platform/crtool/src/js/components/CustomerReviewToolComponent.js [hunk 1]
```diff
@@ -77,6 +77,8 @@ export default class CustomerReviewToolComponent extends React.Component {
      */
     setDistinctiveCompletionDateNow(distinctiveName) {
         let distinctiveCompletionDates =  this.state.distinctiveCompletedDate;
+        console.log("setDistinctiveCompletionDateNow: + " + distinctiveName);
+        console.log(distinctiveCompletionDates[distinctiveName]);
```
Comment (dcmouyard) [inline teachers_digital_platform/crtool/src/js/components/CustomerReviewToolComponent.js]: Remove console.log statements.

File: teachers_digital_platform/jinja2/teachers_digital_platform/crt-start.html [hunk 2]
```diff
@@ -158,11 +158,16 @@ <h2>Curriculum information</h2>
...
                     <div class="m-notification_content">
                         <div class="m-notification_message">
                             <h3 class="h4">Limitations around saving your work</h3>
-                            <p>After you finish each dimension, we recommend you print the summary or save it as a PDF. Learn more about <button class="a-btn a-btn__link" onclick="openSaveWorkModalWindow();return false;">how to save your work.</a></p>
+                            <p>
+                                After you finish each dimension, we recommend you print the summary or save it as a PDF. Learn more about 
+                                <button class="a-btn a-btn__link" onclick="openSaveWorkModalWindow();return false;">how to save your work.
```
Comment (anselmbradford) [inline teachers_digital_platform/jinja2/teachers_digital_platform/crt-start.html]: Should `return false` use `event.preventDefault instead? https://stackoverflow.com/questions/128923/whats-the-effect-of-adding-return-false-to-a-click-event-listener?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

**Pull request merged**